### PR TITLE
Updates and screenshots for latest M365 app support (3rd try)

### PR DESCRIPTION
**Pull request by erikadoyle**
Hopefully SSO token issues have been resolved.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/extend-m365-teams-message-extension.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/63682745-1267-4347-b6ca-df22417b11ee) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/extend-m365-teams-personal-tab.md) | 93 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a3e342bd-1273-4280-b66c-f40fb8fe40ac) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/edbd1848-b9e9-4fcf-97c9-3db445dba026) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/prerequisites.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e4e7665-d6e3-44c8-95b9-fe56ab412024) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/extend-m365-teams-message-extension.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/45a6ab82-f3e3-4c8c-81b2-29fadd499b6d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/extend-m365-teams-personal-tab.md) | 93 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/22de0e14-c948-4599-b0b2-b4e9473e340f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/48873ffc-0c34-48c5-bae0-a8764cb40ad2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/prerequisites.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8f4a6118-fc7b-4387-a08d-899cca5853aa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/extend-m365-teams-message-extension.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e63e3985-c7a0-46a2-96b1-d3542fd03e89) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/extend-m365-teams-personal-tab.md) | 93 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/262e3e17-bfbd-4a41-af79-b46437d5b876) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/00604302-d7cd-4bca-91dc-9e0533f3badc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/prerequisites.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/73e3e768-8026-41dd-9580-558905e17a61) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/extend-m365-teams-message-extension.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/856eb030-3049-4f92-a8fa-c8f9ed2cf33c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/extend-m365-teams-personal-tab.md) | 93 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/28ff8e61-6736-4a7c-b57d-841c7d728adc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e8e05da4-ce85-4f7f-8bdc-dd9e78f503c0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/prerequisites.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1075d177-ad6f-4913-9fc6-856bcf950f0e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/extend-m365-teams-message-extension.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/578f2b69-797e-469e-b86b-e3563ed149bd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/extend-m365-teams-personal-tab.md) | 93 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c1ac832c-23d4-4853-9947-486eb6b5bff2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ba0f163d-a175-4465-a665-f32b741ed153) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f3a7760d2eefe1db17841b5642a045b18c134ae8/msteams-platform/m365-apps/prerequisites.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f68064f9-c32e-4cdb-838e-389dac2780fc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/50bc44b4e68949b2aa20ba958a1bf6d4d78caaf8/msteams-platform/m365-apps/extend-m365-teams-message-extension.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/35476c80-8d89-4a88-a066-b8112f514760) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/50bc44b4e68949b2aa20ba958a1bf6d4d78caaf8/msteams-platform/m365-apps/extend-m365-teams-personal-tab.md) | 93 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5341c70d-7174-4791-beca-19fbfb04a90a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/50bc44b4e68949b2aa20ba958a1bf6d4d78caaf8/msteams-platform/m365-apps/overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/62989786-7f08-4b24-9eb6-a01b2a6b3242) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/50bc44b4e68949b2aa20ba958a1bf6d4d78caaf8/msteams-platform/m365-apps/prerequisites.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/46002197-4e82-4c30-b4b2-8a8f8f3b7789) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f1b6b5d2ec93e1f11c8654770df5685d09aeabca/msteams-platform/m365-apps/extend-m365-teams-message-extension.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6018975d-5417-477e-86e9-18bca376e150) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f1b6b5d2ec93e1f11c8654770df5685d09aeabca/msteams-platform/m365-apps/extend-m365-teams-personal-tab.md) | 93 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/999af789-e72e-4988-b717-a3f87bcc9cf3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f1b6b5d2ec93e1f11c8654770df5685d09aeabca/msteams-platform/m365-apps/overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3f35466f-fd0f-49bb-9370-eb8246d98965) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f1b6b5d2ec93e1f11c8654770df5685d09aeabca/msteams-platform/m365-apps/prerequisites.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c909a819-f63d-4322-87a4-cc43d60e1009) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/extend-m365-teams-message-extension.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Extend a Teams messaging extension across Microsoft 365
 description: Here's how to update your search-based Teams messaging extension to run in Outlook
-ms.date: 11/15/2021
+ms.date: 2/11/2022
```
Comment (surbhigupta12) [inline msteams-platform/m365-apps/extend-m365-teams-message-extension.md:4]:
```suggestion
ms.date: 02/11/2022
```

File: msteams-platform/m365-apps/extend-m365-teams-message-extension.md [hunk 2]
```diff
@@ -139,14 +139,17 @@ You're now ready to test your messaging extension running in Outlook on Windows
...
 ### Outlook on the web
 
 To preview your app running in Outlook on the web, log in to [outlook.com](https://www.outlook.com) using credentials for your test tenant. Click on **New message**. Open the **More apps** flyout menu on the bottom of the composition window. Your messaging extension will be listed. You can invoke it from there and use it just as you would while composing a message in Teams.
 
+### Outlook
+
+> [!IMPORTANT]
+> Refer to the latest updates on [Microsoft Teams - Microsoft 365 Developer Blog](https://devblogs.microsoft.com/microsoft365dev/) to check if Outlook on Windows desktop support for Teams personal apps is available to your test tenant.
+
+To preview your app running in Outlook on Windows desktop, open Outlook logged in with credentials for your test tenant. Click on **New Email**. Open the **More apps** flyout menu on the top ribbon. Your messaging extension will be listed. You can invoke it from there and use it just as you would while composing a message in Teams.
```
Comment (surbhigupta12) [inline msteams-platform/m365-apps/extend-m365-teams-message-extension.md:151]: Please convert this into a procedure.

**To preview your app running in Outlook on Windows desktop**

1. Access Outlook.
2. Log in with your test tenant credentials.
3. Click **New Email**.
.....

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Extend a Teams personal tab app across Microsoft 365
 description: Extend a Teams personal tab app across Microsoft 365
-ms.date: 11/15/2021
+ms.date: 2/11/2022
```
Comment (surbhigupta12) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:4]:
```suggestion
ms.date: 02/11/2022
```

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 4]
```diff
@@ -183,14 +183,21 @@ To view your app in Outlook on the web, visit https://outlook.office.com and sig
 
 Click on your app icon to launch and preview your app running in Outlook on the web.
 
-### Office on the web
+### Office
 
-> [!IMPORTANT]
-> Refer to the latest updates on [Microsoft Teams - Microsoft 365 Developer Blog](https://devblogs.microsoft.com/microsoft365dev/) to check if Office.com support for Teams personal apps is available to your test tenant.
+To view your app running in Office on Windows desktop, launch Office and sign in using your dev tenant account. Click on the ellipses (**...**) on the side bar. Your sideloaded app title will appear among your installed apps.
```
Comment (surbhigupta12) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:188]: Suggest converting into a procedure

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 5]
```diff
@@ -183,14 +183,21 @@ To view your app in Outlook on the web, visit https://outlook.office.com and sig
...
+:::image type="content" source="images/office-desktop-more-apps.png" alt-text="Click on the ellipses ('More apps') option on the side bar of Office desktop client to see your installed personal tabs":::
+
+Click on your app icon to launch your app in Office.
+
+### Office on the web
 
 To preview your app running in Office on the web, log into office.com with test tenant credentials. Click on the ellipses (**...**) on the side bar. Your sideloaded app title will appear among your installed apps.
 
-Click on your app icon to launch your app in Office Home.
+:::image type="content" source="images/office-web-more-apps.png" alt-text="Click on the ellipses ('More apps') option on the side bar of office.com to see your installed personal tabs":::
```
Comment (surbhigupta12) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:198]: same as above

File: msteams-platform/m365-apps/overview.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Extend Teams apps across Microsoft 365 (preview)
 description: Extend your Teams app experiences to other high-usage areas of Microsoft 365 
-ms.date: 11/15/2021
+ms.date: 2/11/2022
```
Comment (surbhigupta12) [inline msteams-platform/m365-apps/overview.md:4]:
```suggestion
ms.date: 02/11/2022
```

File: msteams-platform/m365-apps/prerequisites.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Set up your dev environment for extending Teams apps across Microsoft 365
 description: Here are the prerequisites for extending your Teams apps across Microsoft 365
-ms.date: 11/15/2021 
+ms.date: 2/11/2022
```
Comment (surbhigupta12) [inline msteams-platform/m365-apps/prerequisites.md:4]:
```suggestion
ms.date: 2/11/2022
```

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Minor comments. Please verify.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6dc68b8544dcb9c4d1717e7c582903f0efec7b37/msteams-platform/m365-apps/extend-m365-teams-message-extension.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f50b1a86-b663-4fc4-9e00-3187ad7a23fd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6dc68b8544dcb9c4d1717e7c582903f0efec7b37/msteams-platform/m365-apps/extend-m365-teams-personal-tab.md) | 93 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0fc3733-bfbf-43f6-b6b4-7aad5b7b364e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6dc68b8544dcb9c4d1717e7c582903f0efec7b37/msteams-platform/m365-apps/overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2b80dacf-f5a2-4a1e-af73-81229871fce3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6dc68b8544dcb9c4d1717e7c582903f0efec7b37/msteams-platform/m365-apps/prerequisites.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0981182b-e893-4b5c-97ce-95681abe8913) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/extend-m365-teams-message-extension.md (hunk 1 above)
Comment (erikadoyle) [reply msteams-platform/m365-apps/extend-m365-teams-message-extension.md]: Fixed

File: msteams-platform/m365-apps/extend-m365-teams-message-extension.md (hunk 2 above)
Comment (erikadoyle) [reply msteams-platform/m365-apps/extend-m365-teams-message-extension.md]: Done

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md (hunk 3 above)
Comment (erikadoyle) [reply msteams-platform/m365-apps/extend-m365-teams-personal-tab.md]: Fixed

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md (hunk 4 above)
Comment (erikadoyle) [reply msteams-platform/m365-apps/extend-m365-teams-personal-tab.md]: Done

File: msteams-platform/m365-apps/overview.md (hunk 6 above)
Comment (erikadoyle) [reply msteams-platform/m365-apps/overview.md]: Fixed

File: msteams-platform/m365-apps/prerequisites.md (hunk 7 above)
Comment (erikadoyle) [reply msteams-platform/m365-apps/prerequisites.md]: Fixed

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md (hunk 5 above)
Comment (erikadoyle) [reply msteams-platform/m365-apps/extend-m365-teams-personal-tab.md]: Done

**Comment by erikadoyle** [comment]
Thanks @surbhigupta12! Let me know if there's anything else

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**